src/hardware/driver/mod.rs

```diff
@@ -51,3 +53,21 @@ impl ChannelVariant {
         }
     }
 }
+
+#[derive(
+    Clone, Copy, Debug, Default, PartialEq, Eq, Serialize, Deserialize,
+)]
+pub enum LaserInterlock {
+    Asserted,
```

Comment:
I'd hesitate calling an interlock that is in it's "not tripped" state "asserted".
Propose `NotTripped` to resolve ambiguities.